airflow/providers/microsoft/azure/hooks/wasb.py

```diff
@@ -57,6 +57,50 @@ class WasbHook(BaseHook):
     conn_type = 'wasb'
     hook_name = 'Azure Blob Storage'
 
+    @staticmethod
+    def get_connection_form_widgets() -> Dict[str, Any]:
+        """Returns connection widgets to add to connection form"""
+        from flask_appbuilder.fieldwidgets import BS3TextFieldWidget
+        from flask_babel import lazy_gettext
+        from wtforms import StringField
+
+        return {
+            "extra__wasb__connection_string": StringField(
+                lazy_gettext('Blob Storage Connection String (optional)'), widget=BS3TextFieldWidget()
+            ),
+            "extra__wasb__shared_access_key": StringField(
+                lazy_gettext('Blob Storage Shared Access Key (optional)'), widget=BS3TextFieldWidget()
+            ),
+            "extra__wasb__tenant_id": StringField(
+                lazy_gettext('Tenant Id (Active Directory Auth)'), widget=BS3TextFieldWidget()
+            ),
+            "extra__wasb__sas_token": StringField(
+                lazy_gettext('SAS Token (optional)'), widget=BS3TextFieldWidget()
+            ),
+        }
+
+    @staticmethod
+    def get_ui_field_behaviour() -> Dict:
+        """Returns custom field behaviour"""
+        return {
+            "hidden_fields": ['schema', 'port', 'host'],
+            "relabeling": {
+                'login': 'Blob Storage Login (optional)',
+                'password': 'Blob Storage Token (optional)',
```

Comment:
```suggestion
                'password': 'Blob Storage Key (optional)',
```
Should we use `Key` instead of `Token`? Considering that users might mistake it for SAS token?

Reply:
I agree I'll make the change